src/types.go

```diff
@@ -34,22 +34,26 @@ type MatchWithKey struct {
 
 // UserProfile wrapper for datastore
 type UserProfile struct {
-	Tournament string
-	Name       string
-	Rating     float64
-	Wins       int
-	Losses     int
-	JoinDate   time.Time
+	Tournament     string
+	Name           string
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
+	Wins           int
+	Losses         int
+	JoinDate       time.Time
 }
 
 // UserProfileToShow wrapper for datastore
 type UserProfileToShow struct {
-	Name    string
-	Rating  float64
-	FFAWins int
-	Wins    int
-	Losses  int
-	Badges  []Badge
+	Name           string
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
+	FFAWins        int
+	Wins           int
+	Losses         int
+	Badges         []Badge
```

Comment:
You can use a new struct to hold new data.
We don't want to break current table.

Reply:
I made a test run and it seems work fine: https://8080-dot-5290404-dot-devshell.appspot.com/?authuser=1

And it's adding new fields on existing table, which should be ok.